### PR TITLE
26160: added `Source.empty` in Java API

### PR DESCRIPTION
**Pull request by Seetaramayya**
added `Source.empty` which take Class as a parameter and produce empty source (in Java API)

### CODE REVIEW

File: akka-stream/src/main/scala/akka/stream/javadsl/Source.scala [hunk 1]
```diff
@@ -40,6 +40,11 @@ object Source {
    */
   def empty[O](): Source[O, NotUsed] = _empty.asInstanceOf[Source[O, NotUsed]]
 
+  /**
+    * Create a `Source` with no elements. This is an alias to `def empty[O]()`
```
Comment (johanandren) [inline akka-stream/src/main/scala/akka/stream/javadsl/Source.scala]: I think it should rather say `The result is the same as calling Source.<O>empty()`. (Avoid Scala syntax  in javadocs + the concept of alias means something else for Scala)

**Review by johanandren — COMMENTED** [review]
Good addition, a small comment about the java docs

**Comment by raboof** [comment]
Refs #26160

File: akka-stream/src/main/scala/akka/stream/javadsl/Source.scala (hunk 1 above)
Comment (Seetaramayya) [reply akka-stream/src/main/scala/akka/stream/javadsl/Source.scala]: Yes I got your point, corrected the documentation

**Review by johanandren — APPROVED** [review]
LGTM

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
Failure is formatting related, just recompile locally and it will trigger a reformat and commit-push that.

**Comment by Seetaramayya** [comment]
I thought documentation changes does not fail the build, it did :( 
Sorry for that. I hope I'll not repeat this mistake again

**Pull request merged**